Libraries/Animated/Interpolation.js

```diff
@@ -208,7 +208,7 @@ function createInterpolationFromStringOutputRange(
     // ->
     // 'rgba(${interpolations[0](input)}, ${interpolations[1](input)}, ...'
     return outputRange[0].replace(stringShapeRegex, () => {
-      return String(interpolations[i++](input));
+      return String(Math.round(interpolations[i++](input)));
```

Comment:
Hmm actually this is not safe. It will also round the alpha value, so that 0.5 will be rounded to 1 :(